src/services/order.js

```diff
@@ -0,0 +1,94 @@
+const { Order, OrderItem, Product } = require('../config');
+const cartService = require('./cart');
+
+class OrderService {
+  async createOrder(data) {
+    //Get the cart to get the CartItems and total
+    const cart = await cartService.findCartbyUserId(data.UserId);
+
+    //We verify that the cart has items on it
+    if (cart.total === 0 && cart.CartItems.length === 0) {
+      //If not, then we throw the error.
+      throw new Error('Cart has no items');
+    } else {
+      //If it has items, then we create the order adding the User id as FK
+      const order = Order.build({
+        status: 'confirmed',
```

Comment:
I think the first state should be something like 'awaiting payment' ? and this should change when we the user 'pay' the order? Now I think we need a pay/checkout service 🤔 we haven't thought about this, right?